README.md

```diff
@@ -15,10 +15,25 @@
 A distributed ledger for ecology built on top of the
 [cosmos-sdk](http://github.com/cosmos/cosmos-sdk).
 
-## Getting Started
+## Quickstart
 
+You'll need Go 1.17 installed.
+
+```bash
+git clone https://github.com/regen-network/regen-ledger/
+cd regen-ledger
+git checkout v1.0.0
+make install
+regen init choose-cool-name-and-put-here
+wget wget https://github.com/regen-network/mainnet/raw/main/regen-1/genesis.json
+cp genesis.json ~/.regen/config
+regen start --p2p.persistent_peers 69975e7afdf731a165e40449fcffc75167a084fc@104.131.169.70:26656
+```
```

Comment:
This runs a full-node on mainnet, right? So right after running this, the user needs to wait X hours to sync?

IMO a better UX for new-joiners reading this README would be to run a local node where they can play with test transactions.